mobile/native/src/event/api.rs

```diff
@@ -125,6 +129,7 @@ impl Subscriber for FlutterSubscriber {
             EventType::ServiceHealthUpdate,
             EventType::ChannelStatusUpdate,
             EventType::BackgroundNotification,
+            EventType::FundingChannelNotification,
```

Comment:
I only spent 1 hour yesterday debugging because I forgot to add by new `EventType` ❤️

Reply:
Sorry to hear ... this literally happened to me as well :/